### PR TITLE
9087 defer ownable LCA/ICA accounts

### PR DESCRIPTION
**Pull request by turadg**
refs: #9087 
stacked on: https://github.com/Agoric/agoric-sdk/pull/9415

## Description

Ownabality/tradability of LCA/ICA accounts (#9087) is not a requirement for the next release. When it is needed, we can retrofit with Exo class upgrades.

This PR removes the stubs so they don't appear to be needing implementation.

It also has a bunch of other cleanup done in the course of exploring.

### Security Considerations

Takes away a hypothetical authority.

### Scaling Considerations

none

### Documentation Considerations

Adds some new documentation on the Exo relationship. That could get stale but I think it's worth having.

### Testing Considerations

CI

### Upgrade Considerations

Not yet deployed.

### CODE REVIEW

**Review by dckc — COMMENTED** [review]
the relevant commits look pretty good

I'm inclined to wait until the stacking settles down to approve

**Review by dckc — COMMENTED** [review]
the telescope bump and codegen commits already landed, right?

Why are they still in this PR? Why does this PR still change 150+ files?

**Comment by turadg** [comment]
>the telescope bump and codegen commits already landed, right?

> Why are they still in this PR?

Apparently Github didn't update the base _commit_ within master when it did https://github.com/Agoric/agoric-sdk/pull/9420#event-12976260562. I'll manually rebase.

**Comment by turadg** [comment]
Removed @dtribble from reviewers because he confirmed offline that we can defer ownables as long as we'll be able to fast follow with an upgrade adding the ability

**Review by dckc — APPROVED** [review]

**Pull request merged**
